jellyfin_kodi/database/__init__.py

```diff
@@ -392,6 +392,15 @@ def get_credentials():
 
     credentials['Servers'] = credentials.get('Servers', [])
 
+    # Migration for #145
+    # TODO: CLEANUP for 1.0.0 release
+    for server in credentials['Servers']:
+        if 'LocalAddress' in server:
+            server['address'] = server['LocalAddress']
+            del server['LocalAddress']
```

Comment:
We should probably take a user defined address over the autodetected.  Maybe delete `LocalAddress` if `ManualAddress` exists too?
```
if 'ManualAddress' in server:
    server['address'] = server['ManualAddress']
    del server['ManualAddress']
elif 'LocalAddress' in server:
    server['address'] = server['LocalAddress']
    del server['LocalAddress']
```
I just tested and this worked for me

Reply:
If you're adding a check for LocalAddress might as well include RemoteAddress as a final fallback.

Reply:
Fair point.  We pulled the remote address out back in august with the wan IP in the server, but i'm sure some people have installs from that long that can still have it in the config.